cumulusci/cumulusci.yml

```
@@ -1187,6 +1190,13 @@ services:
         description: The email address to used by Github tasks when an operation requires an email address.
         required: True
     validator: cumulusci.core.github.validate_service
+  marketing-cloud:
+    description: Configure a connection to a Marketing Cloud instance
+    class_path: cumulusci.core.config.MarketingCloudServiceConfig
+    attributes:
+      oauth_client:
```

```suggestion
      oauth2_client:
```

Let's always say OAuth2 rather than OAuth, because OAuth1 is a pretty different spec.